game/combo.go

```diff
@@ -48,6 +48,7 @@ type Board interface {
 	Get(Position) (Square, error)
 	AvailableMoves(Color) []Move
 	IfMove(Move) Board
+	RemainingPieces(Color) int
```

Comment:
Yea, I wasn't sure what made the most sense so I just copied what you did with AvailableMoves.  Should I move them both off or just RemainingPieces?